docs/src/main/asciidoc/spanner.adoc

```diff
@@ -25,51 +27,180 @@ dependencies {
 }
 ----
 
+This setup takes care of bringing in the latest compatible version of Cloud Java Spanner libraries as well.
+
+== Spring Data Spanner
+
+
+=== Configuration
+
+To setup Spring Data Spanner, you have to configure connection to Cloud Spanner and Repositories (if you want to use them).
+
+==== Spanner settings
+
+You will need to use link:../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Boot Starter for Spring Data Spanner] to configure Google Cloud Spanner in your Spring application. It contains all the necessary setup that makes it easy to authenticate with your Google Cloud project.
+The following configuration options are available:
+
+|===
+| Name | Description | Optional | Default value
+| `spring.cloud.gcp.spanner.projectId` | the GCP project ID to use | Yes | provided by Spring Cloud GCP
```

Comment:
Please use **project-id** connotation, for this and the other properties.

Reply:
okay, I didn't know that either! Are they equivalent?

Reply:
Yep. Take a look at https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-relaxed-binding for other options